plasma_framework/contracts/src/framework/BlockController.sol

```diff
@@ -84,4 +84,8 @@ contract BlockController is Operated, VaultRegistry {
         nextDepositBlock++;
         return blknum;
     }
+
+    function getDepositBlockNumber() public view returns (uint256) {
```

Comment:
minor: make sure this is covered by js unit tests too (or leave issue to track this)

Reply:
Minor: meanwhile, we are exposing both `nextDepositBlock` and this `getDepositBlockNumber`. To me, it seems pretty confusing what is the different. Probably we should limit to expose one of them only?

Reply:
this is taken from RootChain. 
`nextDepositBlock` has values from range 1-999, which enumerates deposits between child chain blocks
On the other hand `getDepositBlockNumber` is the number of the actual block the deposit is stored in.

Maybe change name of `nextDepositBlock` into sth like `nextDeposit, by removing `block` from var's name we will get rid of confusion